graylog2-web-interface/src/components/errors/ReportedError.test.jsx

```diff
@@ -50,25 +50,24 @@ describe('ReportedError', () => {
   });
 
   it('displays runtime error page when react error got reported', async () => {
-    const { getByText } = render(<ReportedError router={router}>Hello World!</ReportedError>);
+    const { getByText, queryByText } = render(<ReportedError router={router}>Hello World!</ReportedError>);
 
     suppressConsole(() => {
       ErrorsActions.report(createReactError(new Error('The error message'), { componentStack: 'The component stack' }));
     });
-
-    await waitForElementToBeRemoved(() => getByText('Hello World!'));
+    await wait(() => expect(queryByText('Hello World!')).toBeNull());
     await wait(() => expect(getByText('Something went wrong.')).not.toBeNull());
     await wait(() => expect(getByText('The error message')).not.toBeNull());
   });
 
   it('displays unauthorized error page when unauthorized error got reported', async () => {
-    const { getByText } = render(<ReportedError router={router}>Hello World!</ReportedError>);
+    const { getByText, queryByText } = render(<ReportedError router={router}>Hello World!</ReportedError>);
 
     suppressConsole(() => {
       ErrorsActions.report(createUnauthorizedError(new FetchError('The request error message', new Error('The request error message'))));
     });
 
-    await waitForElementToBeRemoved(() => getByText('Hello World!'));
+    await wait(() => expect(queryByText('Hello World!')).toBeNull());
```

Comment:
After talking with @linuspahl, he chose `queryByText` because 
> I had to use queryByText  because getByText  would throw “Unable to find an element with the text:”

After looking into [`queryBy` and `getBy`](https://testing-library.com/docs/react-testing-library/cheatsheet) it appears to me we should be using `queryBy` in these other tests as well since `getBy` will never return null, so the test will always succeed in `not.toBeNull()`